modules/ztp-image-based-upgrade-upgrade.adoc

```diff
@@ -128,4 +128,60 @@ cgu-ibu-finalize        30h   Completed   All clusters are compliant with all th
 cgu-ibu-prep            31h   Completed   All clusters are compliant with all the managed policies
 cgu-ibu-upgrade         31h   Completed   All clusters are compliant with all the managed policies
 ----
---
\ No newline at end of file
+--
+
+. You can remove the OADP Operator and its configuration files after a successful upgrade.
```

Comment:
Global change. All OADP reference should be converted to the existing attributes:

![Screenshot from 2024-07-31 11-43-45](https://github.com/user-attachments/assets/67fc9f14-4550-40c7-8204-45d6d2ab00b5)

Reply:
From common-attributes.adoc